app/export.go

```diff
@@ -63,26 +61,28 @@ func (app *UmeeApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs [
 		if err != nil {
 			panic(fmt.Sprintf("failed to decode validator address: %s", err))
 		}
-
 		allowedAddrsMap[addr] = true
 	}
 
+	/* Just to be safe, assert the invariants on current state. */
 	app.CrisisKeeper.AssertInvariants(ctx)
 
+	/* Handle fee distribution state. */
+
 	// withdraw all validator commission
 	app.StakingKeeper.IterateValidators(ctx, func(_ int64, val stakingtypes.ValidatorI) (stop bool) {
 		_, err := app.DistrKeeper.WithdrawValidatorCommission(ctx, val.GetOperator())
 		if err != nil {
-			panic(fmt.Sprintf("failed to withdraw validator commission: %s", err))
+			log.Fatalf("failed to withdraw validator commission: %s", err)
```

Comment:
Define a pattern if we are going to use `log.Fatalf` instead of `panic` for fatal errors 

- We could also use `app.Logger()`